### DISCUSSION TITLE
Onnx to Tensorflow

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/onnxruntime#16974] [Other Q&A]
What is the recommended way or software to perform this conversion that you talk about in the readme ?

**Comment by carzh** [comment]
Hi, which readme are you referring to? 

I only know of this option: [Tensorflow Backend for ONNX](https://github.com/onnx/onnx-tensorflow)

**Reply by ghost** [reply]
Me too, but that is unmaintained.

**Reply by carzh** [reply]
Ah unfortunately, I don't know if there's a better option. You might be able to find some third-party library that does the conversion, but I haven't come across any -- or you might have a little more luck if you ask in one of the ONNX repos?

**Reply by ghost** [reply]
No big deal, I don't really need it now, but thought there could be some package and I was missing it. I guess it's good to remember, and avoid getting to this closed road.

Thanks for your reply.
